corporate/lib/stripe.py

```diff
@@ -4768,6 +4768,25 @@ def invoice_plans_as_needed(event_time: Optional[datetime] = None) -> None:
                     plan.invoice_overdue_email_sent = True
                     plan.save(update_fields=["invoice_overdue_email_sent"])
                 continue
+            elif (
+                plan.fixed_price is not None
+                and not plan.reminder_to_review_plan_email_sent
+                and plan.end_date is not None  # for mypy
+                and plan.end_date - plan.next_invoice_date <= timedelta(days=62)
+            ):
+                context = {
+                    "billing_entity": billing_session.billing_entity_display_name,
+                    "end_date": plan.end_date.strftime("%Y-%m-%d"),
+                    "support_url": billing_session.support_url(),
+                }
+                send_email(
+                    "zerver/emails/reminder_to_review_plan",
+                    to_emails=[BILLING_SUPPORT_EMAIL],
+                    from_address=FromAddress.tokenized_no_reply_address(),
+                    context=context,
+                )
+                plan.reminder_to_review_plan_email_sent = True
+                plan.save(update_fields=["reminder_to_review_plan_email_sent"])
```

Comment:
We probably need code to set this flag back to `False` through whatever mechanism we expect end dates to be mutated/extended. (Presumably, if they move onto a new plan at the end date, it's fine, but if we extend the end date another yet, we wouldn't find out)

Reply:
Set it to `False` in `update_end_date_of_current_plan`